app/database/models/server/thread.ts

```diff
@@ -31,7 +31,7 @@ export default class ThreadModel extends Model implements ThreadModelInterface {
         [THREAD_PARTICIPANT]: {type: 'has_many', foreignKey: 'thread_id'},
 
         /** A THREAD can have multiple THREADS_IN_TEAM. (relationship is 1:N)*/
-        [THREADS_IN_TEAM]: {type: 'has_many', foreignKey: 'team_id'},
+        [THREADS_IN_TEAM]: {type: 'has_many', foreignKey: 'thread_id'},
```

Comment:
If this changes, then you should also update the association on https://github.com/koox00/mattermost-mobile/blob/8f3a1354e4e67378db7537e6a937ce92ddf99e4d/app/database/models/server/thread_in_team.ts#L31 so that it points `thread_id`.

Reply:
Once confirmed, I will update my PR #6119

Reply:
oh! thanks for the catch <3